packages/react-table/src/components/Table/TreeRowWrapper.tsx

```diff
@@ -11,15 +12,27 @@ export const TreeRowWrapper: React.FunctionComponent<RowWrapperProps> = ({
   row,
   ...props
 }: RowWrapperProps) => {
-  const { 'aria-level': level, 'aria-posinset': posinset, 'aria-setsize': setsize, isExpanded, isHidden } = row.props;
+  const {
+    'aria-level': level,
+    'aria-posinset': posinset,
+    'aria-setsize': setsize,
+    isExpanded,
+    isDetailsExpanded,
+    isHidden
+  } = row.props;
   return (
     <Tr
       aria-level={level}
       aria-posinset={posinset}
       aria-setsize={setsize}
       aria-expanded={!!isExpanded}
       isHidden={isHidden}
-      className={css(className, isExpanded && 'pf-m-expandable', isExpanded && styles.modifiers.expanded)}
+      className={css(
+        className,
+        isExpanded && 'pf-m-expandable',
```

Comment:
@nicolethoen  is this class not in the style sheet?  If not,  @mcoker should this be added?

Reply:
Sometimes classes are just placeholders for the future and those don't appear in the styles object. Not sure if this is the case here.

Reply:
yeah - it's a class name in the core example that does not have any styles attached, so they are not in the styles object

Reply:
I don't see `pf-m-expandable` referenced on any table row in core. Also that would typically apply to a row if it's expandable, not just in the expanded state. Should be safe to remove.